packages/snaps-simulator/package.json

```diff
@@ -1,35 +1,14 @@
 {
   "name": "@metamask/snaps-simulator",
   "version": "2.4.3",
+  "private": true,
```

Comment:
Note that I added `private: true` here. We previously published this to NPM for `snaps-jest`, but we no longer use it.